vbml/patcher/abc.py

```diff
@@ -39,17 +39,19 @@ def validator(
         """
 
         def decorator(validator_handler: ValidatorType) -> ABCValidator:
-            validator: ABCValidator
+            validator: typing.Optional[ABCValidator] = None
 
-            if isinstance(validator_handler, types.FunctionType):
+            if inspect.isclass(validator_handler):
```

Comment:
why nested ifs? if one fails, error wont be raised
this def needs test coverage!

Reply:
To prevent any other class from being considered as a `callable`.

Reply:
ok i guess i was baffled because it isn't just combined into single condition like `isclass and issubclass`